packages/pluginutils/src/utils/ensureArray.ts

```diff
@@ -0,0 +1,5 @@
+export default function ensureArray<T>(thing: Array<T> | T | undefined | null): Array<T> {
+  if (Array.isArray(thing)) return thing;
+  if (thing == undefined) return []; // eslint-disable-line no-undefined, eqeqeq
```

Comment:
Should the eslint config be updated to allow `== undefined`? It feels like something that would be useful elsewhere too.

Reply:
Let's evaluate this as we go further in converting the repo to TS in the future. In plain JS it's a good rule.